docs/advanced/witness.md

```diff
@@ -0,0 +1,84 @@
+For example, you create a cluster with `harvester-node-0`, `harvester-node-1` and `harvester-node-2`.
+
+Choose the `harvester-node-1` as the witness node when joining the cluster. After the cluster is created, you would see the `harvester-node-1` only has the etcd capability as follows:
+
+```
+NAME↑               STATUS   ROLE                         VERSION               PODS     CPU      MEM    %CPU    %MEM    CPU/A    MEM/A AGE        
+harvester-node-0    Ready    control-plane,etcd,master    v1.27.10+rke2r1         70    1095    10143      10      63    10000    15976 4d13h      
+harvester-node-1    Ready    etcd                         v1.27.10+rke2r1          7     258     2258       2      14    10000    15976 4d13h      
+harvester-node-2    Ready    control-plane,etcd,master    v1.27.10+rke2r1         36     840     6905       8      43    10000    15976 4d13h      
+```
+
+You can see the `harvester-node-1` use less of resource to running as expection. You need to satisfy the three-node requirement, and the promote controller will promote the second and third nodes.
+
+## Workloads on the witness node
+
+The witness node only runs the must workload like etcd. Now the following list is the supported workloads on the witness node.
+- **harvester-node-manager**
+- **cloud-controller-manager**
+- **etcd**
+- **kube-proxy**
+- **rke2-canal**
+- **rke2-multus**
+
+## Upgrade the cluster with witness node
+
+When you upgrade the cluster with a witness node, you can upgrade the cluster as usual. But if you have a degraded volume, the upgrade might fail. You can check the following limitations for the volume robustness.
+
+## Limitations
+
+There were some limitations for the cluster with witness.
+
+- Volume robustness: The witness node does not store any volume data. It means we will miss one replica when the cluster has the witness node due to the Longhorn default replica number (3). The upgrade might failed by the degraded volume. It happens when the cluster only has three nodes, one of which is the witness node.
+
+  To resolve this, you should create a new StorageClass and decrease its replica number from 3 -> 2. Then, set this StorageClass as a default StorageClass.
+  ![new storageclass replica 2](/img/v1.3/advanced/new-storageclass-rep-2.png)
+  ![set to default](/img/v1.3/advanced/set-to-default-sc.png)
+
+  If you already create some volume with the default StorageClass, you can manually decrease the replica counts from 3 -> 2 on Longhorn UI -> **Volume Page** as a workaround. You can get the related info for how to redirect to Longhorn UI from [here](../troubleshooting/harvester/#access-embedded-rancher-and-longhorn-dashboards)
+  ![redirect-to-longhorn-volume-page](/img/v1.3/advanced/redirect-to-longhorn-vol-page.png)
+  ![update-replica-count-to-2](/img/v1.3/advanced/update-replica-2.png)
+
+  After, that the volume robustness will be healthy.
+
```

Comment:
```suggestion
The witness node does not store any data so only two replicas are created for each Longhorn volume in the cluster. However, the default StorageClass `harvester-longhorn` has a replica count value of `3` for high availability. If you use this StorageClass to create volumes, Longhorn is unable to create the configured number of replicas. This results in volumes being marked as **Degraded** on the Longhorn UI.

To resolve this issue, create a new *default* StorageClass with the **Number of Replicas** parameter set to **2**. This ensures that only two replicas are created for each Longhorn volume.
  
![new storageclass replica 2](/img/v1.3/advanced/new-storageclass-rep-2.png)
![set to default](/img/v1.3/advanced/set-to-default-sc.png)

If you already created volumes using the original default StorageClass, you can modify the replica count on the **Volume** screen of the [embedded Longhorn UI](../troubleshooting/harvester/#access-embedded-rancher-and-longhorn-dashboards).

![redirect-to-longhorn-volume-page](/img/v1.3/advanced/redirect-to-longhorn-vol-page.png)
![update-replica-count-to-2](/img/v1.3/advanced/update-replica-2.png)
```

Reply:
A list is not necessary because there is only one limitation. I also removed "Volume robustness" because it distracts from the main point, which is the incorrect replica count.